### PR TITLE
Add IP option to SAN certificate

### LINKED ISSUE #7939: [DOC]Generating self-signed certificates: Using IP address in SAN syntax
**What do you want to do?**
 
- [x] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.**
In the section detailing creating node and client certificates it mentions specifying a SAN. If a user is creating a certificate for a server that only has an IP address then the syntax `DNS:127.0.0.1` will result in a validation error.

We should change this:

```
Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host:
```

To something like this:

```
Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host. Use `IP:` for IP addresses:
```

***Version:**
All versions?

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Closes #7939 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Naarcha-AWS** [comment]
@cwperks: Mind taking a quick look at this one?

**Comment by cwperks** [comment]
@Naarcha-AWS Looks good to me. wdyt about mentioning that it works with IPv6 as well?

File: _security/configuration/generate-certificates.md [hunk 1]
```diff
@@ -115,13 +115,21 @@ openssl req -new -key node1-key.pem -out node1.csr
 For all host and client certificates, you should specify a subject alternative name (SAN) to ensure compliance with [RFC 2818 (HTTP Over TLS)](https://datatracker.ietf.org/doc/html/rfc2818). The SAN should match the corresponding CN so that both refer to the same DNS A record.
 {: .note }
 
-Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host:
+Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host. If you're connecting to a host that only has an IP address, use `IP`:
```
Comment (Naarcha-AWS) [inline _security/configuration/generate-certificates.md:118]:
```suggestion
Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host. If you're connecting to a host that only has an IP address, both IPv4 and IPv6, use the `IP` syntax:
```

File: _security/configuration/generate-certificates.md [hunk 2]
```diff
@@ -115,13 +115,21 @@ openssl req -new -key node1-key.pem -out node1.csr
 For all host and client certificates, you should specify a subject alternative name (SAN) to ensure compliance with [RFC 2818 (HTTP Over TLS)](https://datatracker.ietf.org/doc/html/rfc2818). The SAN should match the corresponding CN so that both refer to the same DNS A record.
 {: .note }
 
-Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host:
+Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host. If you're connecting to a host that only has an IP address, both IPv4 and IPv6, use the `IP` syntax:
```
Comment (natebower) [inline _security/configuration/generate-certificates.md:118]:
```suggestion
Before generating a signed certificate, create a SAN extension file that describes the DNS A record for the host. If you're connecting to a host that only has an IP address, either IPv4 and IPv6, use the `IP` syntax:
```

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS Just a couple of changes. Thanks!

File: _security/configuration/generate-certificates.md [hunk 3]
```diff
@@ -115,13 +115,21 @@ openssl req -new -key node1-key.pem -out node1.csr
 For all host and client certificates, you should specify a subject alternative name (SAN) to ensure compliance with [RFC 2818 (HTTP Over TLS)](https://datatracker.ietf.org/doc/html/rfc2818). The SAN should match the corresponding CN so that both refer to the same DNS A record.
 {: .note }
 
-Before generating a signed certificate, create a SAN extension file which describes the DNS A record for the host:
+Before generating a signed certificate, create a SAN extension file that describes the DNS A record for the host. If you're connecting to a host that only has an IP address, either IPv4 and IPv6, use the `IP` syntax:
```
Comment (natebower) [inline _security/configuration/generate-certificates.md:118]:
```suggestion
Before generating a signed certificate, create a SAN extension file that describes the DNS A record for the host. If you're connecting to a host that only has an IP address, either IPv4 or IPv6, use the `IP` syntax:
```

**Review by natebower — APPROVED** [review]
LGTM with one minor change that I missed 😄

**Pull request merged**